tests/work_orders/04_work_order_submit.json

```diff
@@ -20,21 +20,15 @@
         "requesterSignature": "",
         "inData": [
                    {"index": 1,
-                    "dataHash": "mhash444",
+                    "dataHash": "",
                     "data": "Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1",
                     "encryptedDataEncryptionKey": "",
                     "iv": ""
                    }
         ],
         "outData": [
```

Comment:
Do we need to pass the outData in work order submit request? I don't think it is required. Can you remove this as well.?

Reply:
Ok